src/AllCocktailsPage/AllCocktailsPage.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
+import { getAllCocktails } from "../apiCalls";
+
+export interface AllCocktailsDetails {
```

Comment:
Had to update page as AllCocktails was being used as the state for an array of cocktails - a better defined component name.

Reply:
Yes. Good call! More semantic naming convention